packages/sanity/src/desk/comments/src/components/pte/blocks/MentionInlineBlock.tsx

```diff
@@ -40,10 +39,8 @@ export function MentionInlineBlock(props: MentionInlineBlockProps) {
   return (
     <Tooltip
       portal
-      open={selected}
```

Comment:
@hermanwikner could you please help me confirm this is not changing any previous behaviour? Couldn't find a way to reproduce this "selected" .
Thanks